charts/kubearchive/templates/api_server/api_server.yaml

```diff
@@ -47,7 +47,7 @@ spec:
             - name: POSTGRES_URL
               value: {{ tpl .Values.database.url . }}
```

Comment:
Is there a default URL for Postgres? If not, since you are requiring a Postgres port you will need to require the URL as well.

Reply:
Note that we use `tpl` in there, using `tpl` implies using `required`. If `.Values.database.url` isn't provided, the `helm lint` and `helm template` commands will fail.